### DISCUSSION TITLE
USPS not showing packages to be delivered

### DISCUSSION THREAD

**Discussion by tetra-archos** [moralmunky/Home-Assistant-Mail-And-Packages#887] [Q&A]
I have my delivery notification emails sent to a folder being watched by Mail and Packages, and everything but USPS shows up for the day, but no matter what it doesn't show packages set to be delivered from USPS informed delivery emails. Is anyone else having the same issue?

**Comment by firstof9** [comment]
You have the emails setup like this?
![image](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/assets/1105672/7a3bed21-536a-42af-b2ee-46f308c739cb)

**Reply by tetra-archos** [reply]
Ah I see, I thought the digest email would be processed, I must have missed this step in the documentation. My bad, thank you.
